### PR TITLE
Feat/add gmail option

### PR DESCRIPTION
**Pull request by accimeesterlin**
Adding services to integrate with first time in the app. 

Screen Shot 2019-09-15 at 10 05 24 PM

### CODE REVIEW

File: src/main/components/email/Integrations.jsx [hunk 1]
```diff
@@ -2,21 +2,23 @@ import React from "react";
 import Content from "../common/content/Content";
 import CardComponent from "../../molecules/card/Card";
 
-const IntegrationComponent = () => {
+import { integrateGmail } from '../../../utils/integrationUtils';
+
+const IntegrationComponent = (props) => {
```
Comment (michellbrito) [inline src/main/components/email/Integrations.jsx:7]: Maybe we can change the name? displayEmailsComponent

File: src/main/components/email/Integrations.jsx [hunk 2]
```diff
@@ -2,21 +2,23 @@ import React from "react";
...
   const listOfIntegrations = [
```
Comment (michellbrito) [inline src/main/components/email/Integrations.jsx:8]: maybe a better name would be listOfAvaliableEmails?

**Review by michellbrito — COMMENTED** [review]
Looks good, just made 2 name suggestions in order to be more clear when others read it

File: src/main/components/email/Integrations.jsx (hunk 1 above)
Comment (accimeesterlin) [reply src/main/components/email/Integrations.jsx:7]: This component is not for displaying emails. Instead, it is to start the email integration with either Gmail or any other services out there.

File: src/main/components/email/Integrations.jsx (hunk 2 above)
Comment (accimeesterlin) [reply src/main/components/email/Integrations.jsx:8]: Maybe list of listOfAvalaibleIntegrations

**Pull request merged**